### PR TITLE
update ATT&CK mappings

### PR DESCRIPTION
**Pull request by williballenthin**
thanks to R.E. for lots of (internal) mapping work that led to these clarifications.

### CODE REVIEW

**Comment by williballenthin** [comment]
@dzbeck wouldn't mind having your thoughts on these changes if you notice anything seems weird/wrong.

File: c2/file-transfer/write-and-execute-a-file.yml [hunk 1]
```diff
@@ -4,8 +4,6 @@ rule:
     namespace: c2/file-transfer
     author: moritz.raabe@fireeye.com
     scope: function
-    att&ck:
-      - Command and Control::Ingress Tool Transfer [T1105]
```
Comment (williballenthin) [inline c2/file-transfer/write-and-execute-a-file.yml:8]: we have this under C2, but are we sure that this rule encodes command and control?

File: collection/credit-card/parse-credit-card-information.yml [hunk 2]
```diff
@@ -4,8 +4,6 @@ rule:
     namespace: collection/credit-card
     author: "@_re_fox"
     scope: function
-    att&ck:
-      - Credential Access::Data from Local System [T1005]
```
Comment (williballenthin) [inline collection/credit-card/parse-credit-card-information.yml:8]: CC data may not necessarily be from local system, nor  be credential data.

File: nursery/read-and-send-data-from-client-to-server.yml [hunk 3]
```diff
@@ -4,8 +4,6 @@ rule:
     namespace: c2/file-transfer
     author: william.ballenthin@fireeye.com
     scope: function
-    att&ck:
-      - Exfiltration::Exfiltration Over C2 Channel [T1041]
```
Comment (williballenthin) [inline nursery/read-and-send-data-from-client-to-server.yml:8]: are we sure this is C2?

File: nursery/receive-and-write-data-from-server-to-client.yml [hunk 4]
```diff
@@ -4,8 +4,6 @@ rule:
     namespace: c2/file-transfer
     author: william.ballenthin@fireeye.com
     scope: function
-    att&ck:
-      - Command and Control::Ingress Tool Transfer [T1105]
```
Comment (williballenthin) [inline nursery/receive-and-write-data-from-server-to-client.yml:8]: are we sure this is C2?

File: c2/file-transfer/write-and-execute-a-file.yml (hunk 1 above)
Comment (dzbeck) [reply c2/file-transfer/write-and-execute-a-file.yml:8]: I don't think it's possible to know it's C2... I think something similar can be said for many rules/mappings, but I think there is value in suggesting such objectives/behaviors as possibilities (might be useful to associate confidence values with mappings). Could replace the current att&ck and mbc mapping with mbc micro behaviors (or add for context), which are literal/certain -- File System::Write File [C0052] and Process::Create Process [C0017].

**Review by mr-tz — APPROVED** [review]
Thanks, great changes!

**Pull request merged**